core/src/services/swift/backend.rs

```diff
@@ -221,12 +223,28 @@ impl Access for SwiftBackend {
 
     async fn stat(&self, path: &str, _args: OpStat) -> Result<RpStat> {
         let resp = self.core.swift_get_metadata(path).await?;
+        
+        debug!(
+            "swift: stat operation for path {}, response status: {}",
+            path,
+            resp.status()
+        );
 
-        let status = resp.status();
-
-        match status {
+        match resp.status() {
             StatusCode::OK | StatusCode::NO_CONTENT => {
-                let meta = parse_into_metadata(path, resp.headers())?;
+                let headers = resp.headers();
+                let mut meta = parse_into_metadata(path, headers)?;
+                
+                // 添加日志：记录解析前的原始头部
+                log::debug!("swift: parsing user metadata from headers: {:?}", headers);
+                
+                let user_meta = parse_prefixed_headers(headers, "X-Object-Meta-");
```

Comment:
I'm guessing we need to use `x-object-meta-` here instead.

Reply:
Ok, I'll try to fix it later.